Add DataStax listing

newkek opened the pull request:
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

auto-nylen commented:
Hi @newkek, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit ac064ca).  See below for details:

```
datastax.md: Invalid section: "Company Size".  Expected one of: ["Company blurb","Company size","Remote status","Region","Company technologies","Office locations","How to apply"]
datastax.md: Invalid section: "Office Locations".  Expected one of: ["Company blurb","Company size","Remote status","Region","Company technologies","Office locations","How to apply"]
datastax.md: Invalid section: "Company Technologies".  Expected one of: ["Company blurb","Company size","Remote status","Region","Company technologies","Office locations","How to apply"]
datastax.md: Profile looks complete, but the main readme contains a warning emoji.

4 problems detected
```

Please fix these issues and push a new commit to this pull request.

auto-nylen commented:
Hi @newkek, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 227d2fb).  See below for details:

```
datastax.md: Profile looks complete, but the main readme contains a warning emoji.

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

auto-nylen commented:
Hi @newkek, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 939652b). :heavy_check_mark:

nylen commented on .gitignore:
```diff
@@ -14,6 +14,9 @@ Icon
 
+# IntelliJ IDEA
+.idea
+
```
Please remove this change, the right way to handle IDE-specific ignore files like this is to use a global ignore file that applies to your whole computer: https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore

nylen commented:
This PR looks good (apart from the unneeded change to `.gitignore`), but we were having an issue with the build (fixed in #662). If you can merge or rebase against the latest code from our `master` branch then we can get these changes in.

newkek commented:
@nylen thanks for reviewing, I removed the `.gitignore` commit and rebased against the latest master, however it seems the `netlify` build still fails.

nylen commented:
@newkek it looks like you didn't do a `fetch` before rebasing, the parent commit of your latest commit is from August 29.

newkek commented:
@nylen Please note: it's been updated and the build passes now
